src/compat.c

```diff
@@ -0,0 +1,20 @@
+#include <postgres.h>
+#include <funcapi.h>
+
+/* Old functions that are no longer used but are needed for compatibiliy when
```

Comment:
Argh I think this is actually because timescaledb--0.1.0--0.2.0.sql actually requires it since it CREATES that function. That's really annoying, not sure what to do about that. Maybe these functions are ok. (Though they should maybe raise an error instead of just returning NULL).